### PR TITLE
Remove last occurrences of agent-platform

### PR DESCRIPTION
**Pull request by chouetz**
### What does this PR do?
Remove occurrences of former team

### CODE REVIEW

**Review by KevinFairise2 — APPROVED** [review]

File: test/new-e2e/README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ As a workaround, you can run tests against the version of the agent built in an
 
 ### Installer tests
 
-Installer tests are tests owned by Agent Platform, that mostly test the installation of the agent on different platforms with different methods. These tests are defined in `./tests/agent-platform`.
+Installer tests are tests owned by Agent Build And Releases. They mostly test the installation of the agent on different platforms with different methods. These tests are defined in `./tests/agent-platform`.
```
Comment (chouquette) [inline test/new-e2e/README.md:32]: Oh we own those? TIL :sweat_smile:

**Review by chouquette — APPROVED** [review]

**Review by amenasria — APPROVED** [review]
LGTM !

File: test/new-e2e/README.md (hunk 1 above)
Comment (spencergilbert) [reply test/new-e2e/README.md:32]: I don't think it's unreasonable for us to own these 🤷

**Pull request merged**